api/v1beta1/flinkcluster_validate.go

```diff
@@ -417,10 +417,19 @@ func (v *Validator) validateMemoryOffHeapMin(
 	return nil
 }
 
-func IsJobActive(cluster *FlinkCluster) bool {
+func isJobActive(cluster *FlinkCluster) bool {
```

Comment:
The logic seems to be quite complex, could you add a comment for this  function?

Reply:
I have added isJobTerminated instead.